src/controller/CommissioningWindowOpener.cpp

```diff
@@ -140,7 +159,12 @@ CHIP_ERROR CommissioningWindowOpener::OpenCommissioningWindow(const Commissionin
     return mController->GetConnectedDevice(mNodeId, &mDeviceConnected, &mDeviceConnectionFailure);
 }
 
-CHIP_ERROR CommissioningWindowOpener::OpenCommissioningWindow(const CommissioningWindowVerifierParams & params)
+CHIP_ERROR CommissioningWindowOpener::OpenCommissioningWindow(const CommissioningWindowVerifierParams & params
```

Comment:
If the spec has a different command for opening the commissioning window of a JF, then we should probably do the same here. The macros make the code difficult to read, and in theory it should be possible to put a device into both types of commissioning mode, right?